### DISCUSSION TITLE
Is it possible to disable internalization completely ?

### DISCUSSION THREAD

**Discussion by HongKilDong** [Sylius/Sylius#13136] [Q&A] [answered]
My shop is local, so I don't need all those locales, translations and so on. Can I disable and remove i18n completely and have only one local ?

**Comment by vvasiloi** [comment, marked as answer]
You cannot disable it, but you can use only one locale and remove it from the urls: https://docs.sylius.com/en/1.10/cookbook/shop/disabling-localised-urls.html

**Comment by HongKilDong** [comment]
And can I remove this excessive locales from each item in Sylius ?
![image](https://user-images.githubusercontent.com/142093/134502342-eb7baa2e-2d2e-44be-b171-1d52bce59bcf.png)

**Reply by vvasiloi** [reply]
You can't remove the locales.
At least not from the UI, but you can probably do it directly from the database.
Why did you add them in the first place?

**Reply by HongKilDong** [reply]
Ok, I get it, they are from shop's demo data )

**Comment by Rafikooo** [comment]
I'm closing this discussion because the question has been answered.
